docs/getting-started/installation.rst

```diff
@@ -38,7 +43,7 @@ Anaconda
 .. image:: https://img.shields.io/conda/vn/conda-forge/pyvista.svg?logo=conda-forge&logoColor=white
    :target: https://anaconda.org/conda-forge/pyvista
 
-To install this package with conda run::
+To install this package with Anaconda run::
```

Comment:
Personally I'd favor using the term conda, referring to the actual packaging tool, instead of the distribution/company.